src/Hero6.DesktopGL/Hero6.DesktopGL.csproj

```diff
@@ -15,6 +15,7 @@
     <MonoGameContentBuilderExe Condition=" '$(OS)' != 'Windows_NT' And Exists ('/opt/monogame-pipeline/MGCB.exe') ">/opt/monogame-pipeline/MGCB.exe</MonoGameContentBuilderExe>
     <TargetFrameworkVersion>v4.5.1</TargetFrameworkVersion>
     <TargetFrameworkProfile />
+    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
```

Comment:
I'm not entirely sure why you're doing this change in the first place, this property seems to be related to .NET 4.5.1, but all our projects is using that version, so why is only this single project getting this property?